src/renderers/webxr/WebXRManager.js

```diff
@@ -270,6 +270,8 @@ class WebXRManager extends EventDispatcher {
 
 				scope.dispatchEvent( { type: 'sessionstart' } );
 
+				return glProjLayer || glBaseLayer;
```

Comment:
How about adding a `getWebGLLayer()` method to `WebXRManager` instead?

Reply:
Makes sense and would be more in line in the rest of the API. I'll change it.